src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private int amountBuy = 0;
+    private int amountSupply = 0;
+    private int result;
```

Comment:
```suggestion

```

Reply:
It is bad to keep this data on class level. For example you created a object of WorkWithFile  class, to calculate some data, when you first calculated everything will be ok, but when you will try to calculate again using same object, this object will keep data about previous calculations, and It will not be possible to reuse it. You will need to create new object of WorkWithFile class to make new calculation. Or as another option you need to clear amountBuy, amountSupply and result after each calculation, but I don`t recommend this approach because it is more complicated and less clear then other options